karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Fri Jan 22 2016 15:42:55 GMT+0000 (GMT Standard Time)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
```

Comment:
I think this block should just be:

~~~js
files: [
  "spect/**/*.js"
]
~~~

since:

  1. You only need to specify tests I think since the tests then require any source code they need (Did I just make this up?).
  1. Using a more generic path allows new code/tests to be added more easily.